lua/orgmode/colors/markup_highlighter.lua

```diff
@@ -12,54 +12,63 @@ local markers = {
     hl_name = 'org_bold',
     hl_cmd = 'hi def org_bold term=bold cterm=bold gui=bold',
     nestable = true,
+    spell = nil,
     type = 'text',
   },
   ['/'] = {
     hl_name = 'org_italic',
     hl_cmd = 'hi def org_italic term=italic cterm=italic gui=italic',
     nestable = true,
+    spell = nil,
     type = 'text',
   },
   ['_'] = {
     hl_name = 'org_underline',
     hl_cmd = 'hi def org_underline term=underline cterm=underline gui=underline',
     nestable = true,
+    spell = nil,
```

Comment:
I believe all `spell = nil` can be removed, since it will default to `nil` when it tries to find something on table that does not exist.